### ISSUE TITLE
Banjo-Tooie

### ISSUE THREAD

**Issue by yoholi** [Rinnegatamante/DaedalusX64-vitaGL-Compatibility#50]
# Game summary
- Game name: Banjo-Tooie
- Game version: Europe

# Daedalus X64 summary
- Version: v0.1 dev build

# Issues

when launching the game, daedalus freezes on the rom selection screen, I waited for about 5 minutes but nothing happened, app was still freezing. I tried twice, same issue

# Screenshots
![](https://?)

# Recommended labels

crash

**Comment by Rinnegatamante** [comment]
Can you try with dynarec disabled?

**Comment by yoholi** [comment]
Yes it does launch with disabled dynarec. About 11-15 fps in menus. It gets in game and has about 5-14 fps (full OC). not playable but I would say ingame+ slow (very slow) without dynarec. I did not crash. I edited my post.

**Comment by josenn** [comment]
![0501201705a](https://user-images.githubusercontent.com/26552123/80845310-6c9d6680-8bce-11ea-9ab4-f2f9831cc6ac.jpg)
I got it to run on both but it gives me a no controller message making it unplayable (I haven't messed with the any controller options in the emu yet if that's a thing)

**Comment by yoholi** [comment]
personally I had no controller issues when I got it to run dynarec disabled, I managed to launch the game
